mkdocs.yml

```diff
@@ -26,6 +26,10 @@ nav:
     - Is EESSI already installed?: getting_access/is_eessi_accessible.md
     - Native: getting_access/native_installation.md
     - Container: getting_access/eessi_container.md
+    - Configuring EESSI:
```

Comment:
How about calling it _Advanced configuration_?

Reply:
And adjust the "Installation" to "Installation and advance configuration"

Reply:
I agree on including the configuration part in the top header, but I wouldn't call it `Advanced`. Sure, you could argue that site-specific LMOD hooks to tune MPI installations are a bit advanced. But e.g. symlinking the GPU drivers is a) super easy and b) very necessary on any GPU capable system. I don't want to imply that readers can just 'skip' this configuration stuff unless they need something _very_ special (and that's what the word 'advanced' seems to imply - that it's not needed for 'regular'  useage. But it is when it comes to GPUs).